GOVERNANCE.md

```diff
@@ -68,10 +83,10 @@ The Buildpack Authors' Tooling team is responsible for helping buildpack authors
 
 * [libcnb](https://github.com/buildpacks/libcnb)
 
-## Emeritus Team
+### Emeritus Team
 Serving as a member of an open source project requires a huge amount of work that cannot be sustained indefinitely. The emeritus team includes members of the project to whom we will always be grateful, but who no longer actively participate in the project.
 
-Project members may graduate to the emeritus team by a supermajority vote from the core team.
+Project members may graduate to the emeritus team by a supermajority vote from Team Leads.
```

Comment:
I think we should also make clear that project members can go emeritus by request, in which case a vote is not required.

Reply:
@ekcasey can you add that?

Reply:
resolved by https://github.com/buildpacks/community/pull/166